Swat/SwatDate.php

```diff
@@ -1,178 +1,169 @@
 
     /**
-     * ADT
+     * ADT.
      */
-    const TZ_DST_SHORT = 4;
+    public const TZ_DST_SHORT = 4;
 
     /**
-     * Alias for {@link SwatDate::TZ_DST_SHORT}
+     * Alias for {@link SwatDate::TZ_DST_SHORT}.
      *
      * @deprecated
      */
-    const TZ_DST_LONG = 5;
+    public const TZ_DST_LONG = 5;
 
     /**
-     * AST/ADT
+     * AST/ADT.
      */
-    const TZ_COMBINED = 6;
+    public const TZ_COMBINED = 6;
 
     /**
-     * AST or ADT, depending on whether or not the date is in daylight time
+     * AST or ADT, depending on whether or not the date is in daylight time.
      */
-    const TZ_CURRENT_SHORT = 7;
+    public const TZ_CURRENT_SHORT = 7;
 
     /**
-     * Alias for {@link SwatDate::TZ_CURRENT_SHORT}
+     * Alias for {@link SwatDate::TZ_CURRENT_SHORT}.
      *
      * @deprecated
      */
-    const TZ_CURRENT_LONG = 8;
-
-    // }}}
-    // {{{ date format constants
+    public const TZ_CURRENT_LONG = 8;
 
     /**
-     * 07/02/02
+     * 07/02/02.
      */
-    const DF_MDY = 1;
+    public const DF_MDY = 1;
 
     /**
-     * 070202
+     * 070202.
      */
-    const DF_MDY_SHORT = 2;
+    public const DF_MDY_SHORT = 2;
 
     /**
-     * July 2, 2002
+     * July 2, 2002.
      */
-    const DF_DATE = 3;
+    public const DF_DATE = 3;
 
     /**
-     * Tuesday, July 2, 2002
+     * Tuesday, July 2, 2002.
      */
-    const DF_DATE_LONG = 4;
+    public const DF_DATE_LONG = 4;
 
     /**
-     * July 2, 2002 10:09 am
+     * July 2, 2002 10:09 am.
      */
-    const DF_DATE_TIME = 5;
+    public const DF_DATE_TIME = 5;
 
     /**
-     * Tuesday, July 2, 2002 10:09 am
+     * Tuesday, July 2, 2002 10:09 am.
      */
-    const DF_DATE_TIME_LONG = 6;
+    public const DF_DATE_TIME_LONG = 6;
 
     /**
-     * 10:09 am
+     * 10:09 am.
      */
-    const DF_TIME = 7;
+    public const DF_TIME = 7;
 
     /**
-     * Aug 5, 2002
+     * Aug 5, 2002.
      */
-    const DF_DATE_SHORT = 8;
+    public const DF_DATE_SHORT = 8;
 
     /**
-     * Aug 5
+     * Aug 5.
      */
-    const DF_DATE_SHORT_NOYEAR = 9;
+    public const DF_DATE_SHORT_NOYEAR = 9;
 
     /**
-     * Aug 5, 2002 10:09 am
+     * Aug 5, 2002 10:09 am.
      */
-    const DF_DATE_TIME_SHORT = 10;
+    public const DF_DATE_TIME_SHORT = 10;
 
     /**
-     * Aug 5, 10:09 am
+     * Aug 5, 10:09 am.
      */
-    const DF_DATE_TIME_SHORT_NOYEAR = 11;
+    public const DF_DATE_TIME_SHORT_NOYEAR = 11;
 
     /**
-     * August 2002
+     * August 2002.
      */
-    const DF_MY = 12;
+    public const DF_MY = 12;
 
     /**
-     * 08 / 2002
+     * 08 / 2002.
      */
-    const DF_CC_MY = 13;
+    public const DF_CC_MY = 13;
 
     /**
-     * 2002
+     * 2002.
      */
-    const DF_Y = 14;
+    public const DF_Y = 14;
 
     /**
-     * 20020822T180526Z
+     * 20020822T180526Z.
      */
-    const DF_ISO_8601_BASIC = 15;
+    public const DF_ISO_8601_BASIC = 15;
 
     /**
-     * 2002-08-22T18:05:26Z
+     * 2002-08-22T18:05:26Z.
```

Comment:
The auto-sentence formatting is messing up these comments. The date will not contain a . but the comment look like it does.